packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -72,9 +70,9 @@ export const getUrlProviderByBlockchain = logInputOutput((chain: EVMBlockchainIn
         throw new Error(`Url provider not found for chain ${chain}`);
     }
     return url;
-}, "getUrlProviderByBlockchain");
+};
 
-export const getBlockExplorerByBlockchain = logInputOutput((chain: EVMBlockchainIncludingTestnet) => {
```

Comment:
why did we remove these? `logInputOutput`

Reply:
Ah, I forgot hit submit on a comment I wrote when I put this PR up:

> To try and clear some cyclical dep build issues I was running into, I dropped logging from these functions.
> 
> Going to leave this change in since logging everything here is probably overkill